src/Badge/Counter.js

```diff
@@ -22,9 +23,18 @@ Counter.displayName = 'Counter';
 
 Counter.propTypes = {
     className: PropTypes.string,
+    localizedText: CustomPropTypes.i18n({
+        ariaLabel: PropTypes.string
+    }),
     notification: PropTypes.bool
 };
 
+Counter.defaultProps = {
+    localizedText: {
+        ariaLabel: 'Unread count'
```

Comment:
I think for this label name `ariaLabel` is too generic and could be used for any `aria-label` on the page. How about something like `counterLabel`? Maybe thats not the right one either but it's a start int he right direction.

Reply:
Makes sense to me 👍